libvirt/provider.go

```diff
@@ -28,8 +28,9 @@ func Provider() terraform.ResourceProvider {
 		},
 
 		DataSourcesMap: map[string]*schema.Resource{
-			"libvirt_network_dns_host_template": datasourceLibvirtNetworkDNSHostTemplate(),
-			"libvirt_network_dns_srv_template":  datasourceLibvirtNetworkDNSSRVTemplate(),
+			"libvirt_network_dns_host_template":  datasourceLibvirtNetworkDNSHostTemplate(),
+			"libvirt_network_dns_srv_template":   datasourceLibvirtNetworkDNSSRVTemplate(),
+			"libvirt_network_dhcp_host_template": datasourceLibvirtNetworkDHCPHostTemplate(),
```

Comment:
Don't you guys find all these templates a bit confusing? Shouldn't all be the same? Just a template where some IPs (with an optional MAC) should be mapped to some names?

Reply:
True, I did not think about using just one template for both dns and dhcp. Once I'll have time to work on this again, I'll try it.